virtual-desktop/package.json

```diff
@@ -28,6 +28,7 @@
     "angular-l10n": "~5.2.0",
     "angular2-template-loader": "~0.6.2",
     "bootstrap": "~4.3.1",
+    "codelyzer": "^5.1.0",
```

Comment:
You're using "^" here instead of "~" which is fine if that's what you intended to do, but just make sure that's what you want it to do

https://dev.to/saurabhdaware/but-what-the-hell-is-package-lock-json-b04

Reply:
Yeah, I realized this later and have changed them to tilde.